test/Lokad.ILPack.Tests/Lokad.ILPack.Tests.csproj

```diff
@@ -1,7 +1,7 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>netcoreapp3.1;net5.0</TargetFrameworks>
+    <TargetFrameworks>netcoreapp3.1;net5.0;net6.0</TargetFrameworks>
```

Comment:
Support for both `netcoreapp3.1` and `net5.0` ends this year https://dotnet.microsoft.com/en-us/platform/support/policy/dotnet-core Please remove those as well.